### PR TITLE
Added show hide methods

### PR DESCRIPTION
**Pull request by mikefrancis**

### CODE REVIEW

File: src/scss/components/_modal.scss [hunk 1]
```diff
@@ -1,15 +1,15 @@
 $modal-margin: ($line-height-base * 2);
 
 .modal {
-    display: none;
+    //display: none;
```
Comment (juanserra) [inline src/scss/components/_modal.scss]: Can we get rid of this comment?

Comment (mikefrancis) [inline src/scss/components/_modal.scss]: Will do, thanks

File: dist/js/modules/progress-button.js [hunk 2]
```diff
@@ -11,21 +11,38 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
...
         _classCallCheck(this, ProgressButton);
 
         this.button = button;
     }
 
+    /**
+     * Handle the button in a loading state.
+     */
+
+
```
Comment (juanserra) [inline dist/js/modules/progress-button.js]: Shouldn't we have the docblock comments right in top of the methods? I mean, with no blank lines? Just asking :)

Comment (mikefrancis) [inline dist/js/modules/progress-button.js]: This is compiled by babel, don't really have any control over this.

Comment (juanserra) [inline dist/js/modules/progress-button.js]: Just ignore me then

**Pull request merged**